build/scripts/build-inspector.sh

```diff
@@ -41,7 +42,10 @@ find "$WEBKIT_BUILD_OUTPUT_PATH/Build/Products/$CONFIGURATION" -name "*.framewor
 
 checkpoint "Building Inspector app..."
 rm -rf "$INSPECTOR_BUILD_OUTPUT_PATH"
+
+version=$(python "$BUILD_DIR/scripts/get_version.py" "$BUILD_DIR/npm/inspector_package.json" 2>&1)
```

Comment:
I decided to reuse the python logic, but it's OK for me to change it if you wish :)